openpype/hosts/houdini/plugins/create/create_review.py

```diff
@@ -66,9 +62,96 @@ def create(self, subset_name, instance_data, pre_create_data):
 
         instance_node.setParms(parms)
 
-        # Lock any parameters in this list
-        to_lock = [
-            "family",
-            "id"
-        ]
+        to_lock = ["id", "family"]
+
         self.lock_parameters(instance_node, to_lock)
+
+    def get_pre_create_attr_defs(self):
+        attrs = super().get_pre_create_attr_defs()
+        image_format_enum = [
+            {
+                "value": "png",
+                "label": ".png"
+            },
+            {
+                "value": "tif",
+                "label": ".tif"
+            },
+            {
+                "value": "sgi",
+                "label": ".sgi"
+            },
+            {
+                "value": "pic.gz",
+                "label": ".pic.gz"
+            },
+            {
+                "value": "rat",
+                "label": ".rat"
+            },
+            {
+                "value": "jpg",
+                "label": ".jpg"
+            },
+            {
+                "value": "cin",
+                "label": ".cin"
+            },
+            {
+                "value": "rta",
+                "label": ".rta"
+            },
+            {
+                "value": "rat",
+                "label": ".rat"
+            },
+            {
+                "value": "bmp",
+                "label": ".bmp"
+            },
+            {
+                "value": "tga",
+                "label": ".tga"
+            },
+            {
+                "value": "rad",
+                "label": ".rad"
+            },
+            {
+                "value": "exr",
+                "label": ".exr"
+            },
+            {
+                "value": "pic",
+                "label": ".pic"
+            }
+        ]
+
+        return attrs + [
+            BoolDef("keepImages",
+                    label="Keep Image Sequences",
+                    default=False),
+            EnumDef("imageFormat",
+                    image_format_enum,
+                    label="Image Format Options"),
+            BoolDef("override_resolution",
+                    label="Override resolution",
+                    tooltip="When disabled the resolution set on the camera "
+                            "is used instead.",
+                    default=True),
+            NumberDef("resx",
+                      label="Resolution Width",
+                      default=1280,
+                      minimum=2,
+                      decimals=0),
+            NumberDef("resy",
+                      label="Resolution Height",
+                      default=720,
+                      minimum=2,
+                      decimals=0),
```

Comment:
Oh boy, I really want to test this - does this actually work? Does this actually set the resolution attribute on the node or does this become a custom attribute?

Reply:
yes it does work.
you can check by your own.

Reply:
Yes, works! I thought these were instance attribute but they are creation attributes which makes them more trivial.
I thought these were "live" keeping sync with the values from the instance, etc.

Thanks.